src/irr/asset/CImageLoaderOpenEXR.cpp

```diff
@@ -41,17 +41,69 @@ namespace irr
 
 			SContext ctx;
 
-			readVersionField(_file, ctx);
-			readHeader(fileName, ctx);
+			// readVersionField(_file, ctx);
+			// readHeader(fileName, ctx);
 
 			Array2D<Rgba> pixels;
 			int width;
 			int height;
 
 			readRgba(fileName, pixels, width, height);
 
-			// TODO put data into buffer, consider OpenEXR 2.0 supported saving layouts 
-			return SAssetBundle{};
+			constexpr uint32_t MAX_PITCH_ALIGNMENT = 8u;									   // OpenGL cannot transfer rows with arbitrary padding
+			auto calcPitchInBlocks = [](uint32_t width, uint32_t blockByteSize) -> uint32_t	   // try with largest alignment first
```

Comment:
this function is being used in more than one image loader, maybe we should make it into a static method of something common?

Reply:
OpenEXR uses it as well, it's a good idea I think